providers/fab/src/airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -729,6 +735,15 @@ def auth_role_admin(self):
         """Get the admin role."""
         return self.appbuilder.get_app.config["AUTH_ROLE_ADMIN"]
 
+    @property
+    @deprecated(
```

Comment:
Why? We are cutting major release we can just remove them and add relevant entry to change log

Reply:
I know ... I can remove the deprecation warning. I hesitated. The reason why is that it was present in AF2, I just cherry-picked some code from AF2 branch and this method was deprecated. But I agree, this feels weird

Reply:
After a second thought I'll remove the deprecation. It does not make sense to have a deprecated method we are not planning to remove (because we cannot).

Reply:
Updated